bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/handler/RoombaHandler.java

```diff
@@ -503,4 +529,49 @@ private void reportProperty(String property, @Nullable String value) {
             updateProperty(property, value);
         }
     }
+
+    @Override
+    public void connectionStateChanged(MqttConnectionState state, @Nullable Throwable error) {
+        if (state == MqttConnectionState.CONNECTED) {
+            MqttBrokerConnection connection = this.connection;
+
+            if (connection == null) {
+                // This would be very strange, but Eclipse forces us to do the check
+                logger.error("Established connection without broker pointer");
+                return;
+            }
+
+            logger.debug("Connection established");
```

Comment:
This is already logged by the framework and could be removed.

Reply:
Forgot, sorry